modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_1_1/test/DDMStructureCTUpgradeProcessTest.java

```diff
@@ -60,15 +60,16 @@ public void setUp() throws Exception {
 	@Override
 	protected CTModel<?> addCTModel() throws Exception {
 		_ddmStructure = DDMStructureTestUtil.addStructure(
-			_group.getGroupId(), DDMFormInstance.class.getName(),
+			_group.getGroupId(), getClassName(),
 			DDMFormTestUtil.createDDMForm(RandomTestUtil.randomString()));
 
-		return _ddmStructure;
+		setDefinition(_ddmStructure);
```

Comment:
Why do you need to use this method here?

Reply:
why did you create this method?

Reply:
Hi @marcelabc !

In order to make the class generic, making it ready for other cases in which a definition update is need and also produce less code I did as shown in the PR. I tested the other tests and everything was okay. By doind what I did I was able to make the tests from v3_7_1 way more clear and with less lines of code wihtout having a bunch of overrides that only changed one line when comparing with the original code.

Reply:
I created a method in this class in order to avoid copying code, since I had two calls of the same code, in order to write fewer lines and make the code more optimized I decided to abstract the logic to a method, which now is called twice as is our recommendation for creating methods ins't it?